src/swarm/node/model/NeoNode.d

```diff
@@ -813,16 +821,15 @@ public class NodeBase ( ConnHandler : ISwarmConnectionHandler ) : INodeBase
         UnixListener unix_listener;
         if ( options.unix_socket_path.length )
         {
-            BasicCommandHandler.Handler[istring] unix_socket_handlers;
             if ( this.credentials_file )
             {
-                unix_socket_handlers =
+                this.unix_socket_handlers =
                     ["update-credentials"[]: &this.handleUpdateCredentials,
                      "list-credentials": &this.handleListCredentials];
             }
 
-            unix_listener = new UnixListener(
-                options.unix_socket_path, options.epoll, unix_socket_handlers);
+            unix_listener = new UnixListener(options.unix_socket_path,
+                options.epoll, this.unix_socket_handlers);
```

Comment:
I think `UnixListener` doesn't make a copy of the request handlers, but keeps a reference to these, so it's safe to do this (because in derived class, you would first create the handler then adjust the `BasicCommandHandler` array).

Reply:
You're confirming that my change is correct, right?

Reply:
Correct.